### PR TITLE
Rails 6

### PR DESCRIPTION
**Pull request by kobsy**
### Summary
This _mostly_ gets everything up-and-running for Rails 6. The remaining wrinkle is a hack applied within an instance to allow Rails to find the correct `config.ru` (that of the instance) and actually boot the server. The hack is reproduced below and should probably find its way into the instance blueprint if we can't figure out a better way of getting around this difficulty:
```ruby
# bin/ruby
...
require "bundler/setup" if File.exist?(ENV["BUNDLE_GEMFILE"])

### HACK ###
require "rails/command"
require "rails/commands/server/server_command"
config_path = File.expand_path("../config.ru", __dir__)
Rails::Command::ServerCommand.class_options[:config].instance_variable_set(:@default, config_path)
### END HACK ###

require "rails/commands"
```

### CODE REVIEW

File: config/initializers/new_framework_defaults_6_0.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Be sure to restart your server when you modify this file.
```
Comment (lukebooth) [inline config/initializers/new_framework_defaults_6_0.rb:1]: I wonder if we need to get in the habit of removing the old defaults? Doing `config.load_defaults 6.0` I think only loads this file and does nothing with the others. Nothing to change with the PR, just thinking out loud.

**Review by lukebooth — APPROVED** [review]
Looks ok to me! Way to shuffle through this upgrade!

**Comment by kobsy** [comment]
I think this is ready to go, @boblail, but GH is still requiring checks from Travis, when this PR switches to use GitHub Actions instead. Is that a change you're good with, and would you be willing to adjust the branch protection settings accordingly? 😅

File: config/cable.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: test
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```
Comment (robrieba) [inline config/cable.yml:9]: Did you chose database `1` intentionally?

Comment (kobsy) [reply config/cable.yml:9]: Generated by Rails as part of the upgrade, I believe. 🤷

**Review by boblail — APPROVED** [review]
- LGTM!
- Upgrading →5.1→5.2→6.0 in one PR? YOLO!! 😆 
- I appreciated the extra context in your commit message for "[skip] Added GitHub Actions tests (15m)"
- I switched GitHub from requiring a pass from Travis to requiring a pass from GitHub actions ✅

**Pull request merged**